docs/src/content/docs/resources/plugins.mdx

```diff
@@ -1,23 +1,10 @@
 ---
-title: Starlight Showcase
-description: Discover sites built with Starlight and community tools that extend Starlight!
+title: Plugins and Integrations
+description: Discover community tools that extend Starlight!
```

Comment:
```suggestion
description: Discover community tools like plugins and integrations that extend Starlight!
```

The reason I'd avoided it before is that in a social image, the title would display plugins and integrations, so I felt it was OK for the description to be different. But if it seems off to not have this in the page description, fine to add it!